packages/boilerplate-generator/boilerplate-generator.js

```diff
@@ -11,11 +11,9 @@ Boilerplate = function (arch, manifest, options) {
   options = options || {};
```

Comment:
This is really more of a question for @benjamn or @abernix but should new code be written with ES6 these days? import instead of Npm.require (it's ok even in packages to use import for built in packages like this one, right?), no var, class, etc? Using mainModule and imports instead of file lists and package-local variables?

And specifically should this refactored class be rewritten to new style or left alone?